### ISSUE TITLE
Make a cutadapt.exe available

### ISSUE THREAD

**Issue by marcelm** [marcelm/cutadapt#481]
Using PyInstaller, I managed to create a single-file cutadapt executable. The file can be downloaded here: <https://github.com/marcelm/cutadapt/actions/runs/280442469>. The link will expire, so we need to find a more permanent solution. The GitHub actions workflow is in the `windows` branch.

**Comment by marcelm** [comment]
The Cutadapt 3.1 exe is available at https://github.com/marcelm/cutadapt/actions/runs/398847341 (until it expires).

**Comment by rhpvorderman** [comment]
I guess you could add it to the releases page? That won't expire.

**Comment by marcelm** [comment]
Good idea. I haven’t worked with releases before, but that should do what I want.
